watertap/costing/watertap_costing_package.py

```diff
@@ -160,99 +50,24 @@ def build_global_params(self):
             doc="Capital annualization factor [fraction of investment cost/year]",
             units=pyo.units.year**-1,
         )
-
-        self.electricity_cost = pyo.Param(
-            mutable=True,
-            initialize=0.07,
-            doc="Electricity cost",
-            units=pyo.units.USD_2018 / pyo.units.kWh,
-        )
-        self.add_defined_flow("electricity", self.electricity_cost)
-
-        self.electrical_carbon_intensity = pyo.Param(
-            mutable=True,
-            initialize=0.475,
-            doc="Grid carbon intensity [kgCO2_eq/kWh]",
-            units=pyo.units.kg / pyo.units.kWh,
-        )
+        self.capital_recovery_factor.expr = self.factor_capital_annualization
```

Comment:
This seems not ideal but I imagine will be rectified once we agree on how we want to handle which parameters to include and how they will be calculated/what their default values will be.

Reply:
Yeah, I agree it is a bit ugly. But it lets us put `add_LCOW` and similar aggregates in the base class.